pkg/services/govmomi/vcenter/clone.go

```diff
@@ -369,6 +371,18 @@ func getDiskSpec(vmCtx *capvcontext.VMContext, devices object.VirtualDeviceList)
 			diskSpecs = append(diskSpecs, additionalDiskConfigSpec)
 		}
 	}
+
+	// Now if we have increased the disk size of any additional disks that were in the template, we can now add new disks
+	// that are present in the additionalDisks list.
+	if len(vmCtx.VSphereVM.Spec.DataDisks) > 0 {
+		additionalDisks, diskErr := createDataDisks(ctx, vmCtx.VSphereVM.Spec.DataDisks, primaryDisk, devices)
+		if diskErr != nil {
+			klog.Errorf("Unable to add additional disks: %v", diskErr)
```

Comment:
don't import klog, use `log := ctrl.LoggerFrom(ctx)` instead in this function.

Reply:
Sorry, I thought I removed these already.  I had others too, but just missed this one.  Thanks!